styles/colors.css

```diff
@@ -0,0 +1,11 @@
+.bg-blue {
+  background-color: blue;
+}
+
+.bg-red {
+  background-color: red;
+}
+
+.bg-green {
+  background-color: green;
+}
```

Comment:
wasn´t the colors.css for your own build variables for colors which you can use in other css rules? beginning with the :root an in the brackets for example --color-dark-blue: #212d40.

Reply:
i placed the colors here in colors.css to try the "app-grid" in the preview window